README.md

```
@@ -81,77 +81,72 @@ The skim project contains several components:
-`sk` from `apt` and receive updates:
 ```sh
-# Import the signing key
 sudo mkdir -p /etc/apt/keyrings
-sudo wget -O /etc/apt/keyrings/skim.asc https://skim-rs.github.io/skim/apt/skim-archive-keyring.asc
-# ...or with curl:
-# curl -fsSL https://skim-rs.github.io/skim/apt/skim-archive-keyring.asc | sudo tee /etc/apt/keyrings/skim.asc > /dev/null
+curl -fsSL https://skim-rs.github.io/skim/apt/skim-archive-keyring.asc | sudo tee /etc/apt/keyrings/skim.asc > /dev/null
+```
 
-# Add the repository
+2. Add the repository
```

markdownlint-cli2 (0.23.0)

[warning] Ordered list item prefix
Expected: 1; Actual: 2; Style: 1/1/1

(MD029, ol-prefix)